target/linux/brcm47xx/image/Makefile

```diff
@@ -460,31 +500,36 @@ endef
 TARGET_DEVICES += linksys-wrt300n-v1
 
 define Device/motorola-wa840g
-  DEVICE_TITLE := Motorola WA840G
+  DEVICE_VENDOR := Motorola
+  DEVICE_MODEL := WA840G
   DEVICE_PACKAGES := kmod-b43 kmod-b43legacy
   $(Device/motorola)
   MOTOROLA_DEVICE := 2
 endef
 TARGET_DEVICES += motorola-wa840g
 
 define Device/motorola-we800g
-  DEVICE_TITLE := Motorola WE800G
+  DEVICE_VENDOR := Motorola
+  DEVICE_MODEL := WE800G
   DEVICE_PACKAGES := kmod-b43 kmod-b43legacy
   $(Device/motorola)
   MOTOROLA_DEVICE := 3
 endef
 TARGET_DEVICES += motorola-we800g
 
 define Device/motorola-wr850g
-  DEVICE_TITLE := Motorola WR850G
+  DEVICE_VENDOR := Motorola
+  DEVICE_MODEL := WR850G
   DEVICE_PACKAGES := kmod-b43 kmod-b43legacy
   $(Device/motorola)
   MOTOROLA_DEVICE := 1
 endef
 TARGET_DEVICES += motorola-wr850g
 
 define Device/netgear-wgr614-v8
-  DEVICE_TITLE := Netgear WGR614 v8
+  DEVICE_VENDOR := Netgear
```

Comment:
I think so far we generally capitalized NETGEAR (if you agree, also do for the following).